dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -393,20 +393,17 @@ def convert_histogram_key_types_to_np(histogram_info: dict):
             self.histogram_methods[key] = convert_histogram_key_types_to_np(
                 self.histogram_methods[key]
             )
-
         # Convert values to correct types
-        if self.min is not None and type(self.min) not in [np.float64, np.int64]:
-            self.min = (
-                np.float64(self.min) if type(self.min) is float else np.int64(self.min)
-            )
-        if self.max is not None and type(self.max) not in [np.float64, np.int64]:
-            self.max = (
-                np.float64(self.max) if type(self.max) is float else np.int64(self.max)
-            )
-        if type(self.sum) not in [np.float64, np.int64]:
-            self.sum = (
-                np.float64(self.sum) if type(self.sum) is float else np.int64(self.sum)
-            )
+        float_int_to_np_conversion = (
+            np.int64 if self.type in ["int", "string", "text"] else np.float64
+        )
+
+        if self.min is not None:
+            self.min = float_int_to_np_conversion(self.min)
+        if self.max is not None:
+            self.max = float_int_to_np_conversion(self.max)
+        if self.sum is not None:
+            self.sum = float_int_to_np_conversion(self.sum)
```

Comment:
since this is causing issues, let's use prev state and just do np.float64. This will be functional after merging int and float. At profile we can alter min and max for output.

```python
# remove unused float_int_to_np_conversion above

if self.min is not None:
    self.min = np.float64(self.min)
if self.max is not None:
    self.max = np.float64(self.max)
if self.sum is not None:
    self.sum = np.float64(self.sum)
```

Reply:
bingo